### ISSUE TITLE
Question about the answer for the W3.5a video quiz question 3

### ISSUE THREAD

**Issue by Nephelite** [nus-cs2103-AY2223S1/forum#37]
The solution for Q3 of the W3.5a video quiz feels wrong to me.

The correct answer is true, meaning we should change our opinions to fit the module coding guidelines given by the professors. However, the video (and the answer description) both say we should hang on to our opinions but still follow the guidelines given to us (i.e we _shouldn't_ change our opinions). 

I feel that this is a direct contradiction. The video and description implies false is correct, but the correct answer is true.

Seeing the solution, I get what the question is trying to ask, but I think it is phrased incorrectly. Actions taken do not reflect our opinions, and just because I agree to follow a coding guideline, it doesn't mean I think it is correct.

**Comment by damithc** [comment]
@Nephelite The correct answer should be `False`. I thought it's a mistake in the settings but when I checked the settings just now it seems the setting is correct (see below). I saved the setting again just in case. Are you still seeing `True` as the correct answer if you retake the question?

![image](https://user-images.githubusercontent.com/1673303/185731010-9e482b4f-9040-4e17-b679-b0d1c7f4161a.png)
